languagepack/installer.xml.in

```diff
@@ -183,6 +209,13 @@ EOF
         <compareText logic="equals" text="${platform_name}" value="windows"/>
       </ruleList>
     </setInstallerVariable>
+
+    <!-- Upgrading? -->
+    <setInstallerVariable>
+      <name>IsUpgrade</name>
+      <value>0</value>
+    </setInstallerVariable>
```

Comment:
@agarwal-kritika this will always set the upgrade to 0, even if it's the installation is an upgrade. I think this is not needed